doc/developer/building-frr-on-ubuntu1804.rst

```diff
@@ -63,50 +64,50 @@ Add frr groups and user
 
 ::
 
-    sudo groupadd -g 92 frr
-    sudo groupadd -r -g 85 frrvty
-    sudo adduser --system --ingroup frr --home /var/run/frr/ \
-       --gecos "FRR suite" --shell /sbin/nologin frr
-    sudo usermod -a -G frrvty frr
+   sudo groupadd -g 92 frr
+   sudo groupadd -r -g 85 frrvty
+   sudo adduser --system --ingroup frr --home /var/run/frr/ \
```

Comment:
We could merge this command with the following and instead of using `adduser` use `useradd` (same family of commands, like the `groupadd` before).

I think the command would look like this:
```shell
sudo useradd -c "FRR suite" -d /var/run/frr -g frr -G frrvty -r -s /sbin/nologin frr
```

Or you could also use the long options version for clarity:
```shell
sudo useradd --comment "FRR suite" --home-dir /var/run/frr \
  --gid frr --groups frrvty --system --shell /sbin/nologin frr
```